digit-health-services/health-delivery-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,47 @@
+logging.pattern.console=%clr(%X{CORRELATION_ID:-}) %clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}
+
+server.servlet.context-path=/health-health-service
+server.port=8093
+app.timezone=UTC
+
+# DATASOURCE CONFIGURATION
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.hibernate.ddl-auto=validate
+
+# FLYWAY CONFIGURATION
+spring.flyway.url=jdbc:postgresql://localhost:5432/postgres
+spring.flyway.user=postgres
+spring.flyway.password=postgres
+spring.flyway.table=public
+spring.flyway.baseline-on-migrate=true
+spring.flyway.outOfOrder=true
+spring.flyway.locations=classpath:/db/migration/main
+spring.flyway.enabled=true
+
+# KAFKA SERVER CONFIGURATIONS
+kafka.config.bootstrap_server_config=localhost:9092
+spring.kafka.consumer.value-deserializer=org.egov.tracer.kafka.deserializer.HashMapDeserializer
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.group-id=health-sync-service
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
+
+# KAFKA CONSUMER CONFIGURATIONS
+kafka.consumer.config.auto_commit=true
+kafka.consumer.config.auto_commit_interval=100
+kafka.consumer.config.session_timeout=15000
+kafka.consumer.config.auto_offset_reset=earliest
+# KAFKA PRODUCER CONFIGURATIONS
+kafka.producer.config.retries_config=0
+kafka.producer.config.batch_size_config=16384
+kafka.producer.config.linger_ms_config=1
+kafka.producer.config.buffer_memory_config=33554432
+#org.egov.detailed.tracing.enabled = false
+
+# FILE STORE SERVICE
+egov.filestore.host=http://localhost:8083
```

Comment:
We can remove unused env variables like filestore service which we may not use in this service

Reply:
updated.